sources/bottlerocket-variant/src/lib.rs

```diff
@@ -52,13 +52,13 @@ pub mod error {
 /// # Variant
 ///
 /// Represents a Bottlerocket variant string. These are in the form
-/// `platform-runtime-[variant_version]-[variant_flavor]`.
+/// `platform-runtime-[variant_version][-variant_flavor]`.
```

Comment:
Typo?
```suggestion
/// `platform-runtime-[variant_version]-[variant_flavor]`.
```

Reply:
No, that was intentional. It's not strictly related to this PR though, so I can just drop it. There isn't always a `variant_flavor`, so I'm just being a little OCD that the pattern provided would be `aws-k8s-1.25` and not `aws-k8s-1.25-`.